### DISCUSSION TITLE
Getting products

### DISCUSSION THREAD

**Discussion by satvik-1203** [amzn/selling-partner-api-models#3130] [General] [answered]
For a project I'm making, I need to access Amazon's API for getting the JSON products, I looked thru the internet and can't find a simple API, instead, it keeps telling me to use PA API. Is there any other way to do this?

**Comment by teddy-codes** [comment, marked as answer]
The PA API is only for affiliates and they will catch you if you use it.

The SP-API is for people who have access to a seller central account and/or vendor central. You should either create a seller central account and pay the $40/month (for the US marketplace) or find an alternative means for this data (probably more expensive than just making a seller central account and paying).

**Reply by satvik-1203** [reply]
thanks, I just closed my PA API account. :))
